src/components/Navigation.tsx

```diff
@@ -20,11 +24,36 @@ const Navigation: FC = () => {
   const [projectName, setProjectName] = useState(
     project && !isLoading ? project.name : "default",
   );
+  const { hasCustomVolumeIso } = useSupportedFeatures();
+  const enablePermissions = enablePermissionsFeature();
+  const [scroll, setScroll] = useState(false);
+  const [openNavMenus, setOpenNavMenus] = useState<Set<AccordionNavMenu>>(
+    new Set(),
```

Comment:
As we have max 2 items in this set, we should better keep it as a array, set is not making this faster with that few elements.

Reply:
updated state to array instead.